src/components/ScenarioParameters/ScenarioParametersHook.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import {
+  useCurrentScenarioData,
+  useScenarioList,
+  useUpdateAndLaunchScenario,
+  useLaunchScenario,
+} from '../../state/hooks/ScenarioHooks';
+import { useDatasetList, useAddDatasetToStore } from '../../state/hooks/DatasetHooks';
+import { useIsDarkTheme } from '../../state/hooks/ApplicationHooks';
+import { useUserAppRoles } from '../../state/hooks/AuthHooks';
+import { useOrganizationId } from '../../state/hooks/OrganizationHooks';
+import { useWorkspaceId } from '../../state/hooks/WorkspaceHooks';
+import { useSolution } from '../../state/hooks/SolutionHooks';
+
+export const useScenarioParameters = () => {
+  const scenarios = useScenarioList();
+  const datasets = useDatasetList();
+  const addDatasetToStore = useAddDatasetToStore();
+  const currentScenario = useCurrentScenarioData();
+  const organizationId = useOrganizationId();
+  const workspaceId = useWorkspaceId();
+  const solution = useSolution();
+  const userRoles = useUserAppRoles();
+  const launchScenario = useLaunchScenario();
+  const updateAndLaunchScenario = useUpdateAndLaunchScenario();
+  const isDarkTheme = useIsDarkTheme();
+
+  return [
```

Comment:
Here we should return a object instead of array because it's a specific hook used by only one component.